src/containers/BooksForm.js

```diff
@@ -19,21 +34,75 @@ const chooseACategory = () => {
   );
 };
 
-const BooksForm = () => (
-  <form action="">
-    <h2>Insert new book</h2>
-    <input
-      placeholder="Enter Book Title"
-      type="text"
-      name="title"
-    />
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      book: {
+        title: '',
+        category: categories[0],
+      },
```

Comment:
This is a local state that is holding just a book, there is no need really to make a book object in the state unless you have other things which you don't.